mule-extensions-api/src/main/java/org/mule/extension/api/introspection/OperationModel.java

```diff
@@ -37,5 +30,4 @@
      * @return a {@link MetadataType} representing the attribute types for the output messages
      */
     MetadataType getAttributesType();
```

Comment:
in that case you're missing the `@Override`... However, I could argue that the javadoc on the base interface is generic and descriptive enough to not need this.... I'm open to accept this however as long as the override annotation is added